wifiphisher/data/phishing-pages/wifi_connect/config.ini

```diff
@@ -2,7 +2,12 @@
 Name: Network Manager Connect
 Description: The idea is to imitate the behavior of the network manager by first showing the browser's "Connection Failed" page and then displaying the victim's network manager window through the page asking for the pre-shared key.
 [context]
-# The variable enterprise is used when we are working with windows enterprise network
+# The variable enterprise is used when we are working with Windows enterprise network
 # enterprise = 1 : we show the form for login of enterprise networks to the victim
 # enterprise = 0 : we show the form for login of the usual networks to the victim 
-enterprise:0
+enterprise: 1
+
+# The variable enterprise is used when we are working with Mac enterprise network
+# mac_enterprise = 1 : we show the form for login of enterprise networks to the victim
+# mac_enterprise = 0 : we show the form for login of the usual networks to the victim 
+mac_enterprise: 1
```

Comment:
We should have only one variable `enterprise`.

If it's set to `True`, then we show the enterprise version of all network managers.